src/model/censusdata/censusdata.js

```diff
@@ -37,9 +39,21 @@ export function reset() {
   categoryCodeLookup = {};
 }
 
-export async function fetchAllDataForGeography(censusDataService, geographyCode) {
+export async function fetchAllDataForGeography(censusDataService, geographyCode, selectedGeography, overwriteCache) {
   const data = await censusDataService.fetchAllDataForGeography(geographyCode);
-  selectedGeographyData.set(data);
+  //if E&W data - call on app initialise
+  if (geographyCode == config.eAndWGeoCode) {
+    englandAndWalesData.set(data);
+  } else if (selectedGeography) {
+    //write data to selectedGeographyData store
+    selectedGeographyData.set(data);
```

Comment:
what happens if both selectedGeography AND overwriteCache are supplied?

Reply:
then it will overwrite selectedGeographyData and then exit the if/else conditions and function.

Reply:
the overwrite happens by default but I think that's fine in this case

Reply:
as discussed offline, some of these routes are for behaviour thats not needed yet, so have been left out.